src/attribute-editor/internal.tsx

```diff
@@ -61,6 +65,18 @@ const InternalAttributeEditor = React.forwardRef(
     const additionalInfoId = useUniqueId('attribute-editor-info');
     const infoAriaDescribedBy = additionalInfo ? additionalInfoId : undefined;
 
+    const prevItemsLength = usePrevious(items.length);
+
+    React.useEffect(() => {
+      if (prevItemsLength && prevItemsLength > items.length) {
+        setRemovalAnnouncement(i18nStrings?.itemRemovedAriaLive);
```

Comment:
If I remove 2 items in a row, it seems the removal announcement will not be made? Because it is static string and the content of the aria-live is not change.

Reply:
I think it is. Because two actions no necessary being very close, users may remove the first item then browse the list to find the second items to remove, then third.... But they may only hear the removal being announced at the very first action. It may lead to confusion of users if the action being successfully done.